### ISSUE TITLE
(pycharm) Feature-Request: Add EAP versions

### ISSUE THREAD

**Issue by mastacheata** [mkevenaar/chocolatey-packages#57]
## Expected Behavior

Can you please publish the EAP builds of PyCharm as prerelease versions to chocolatey?

## Current Behavior

Currently only final versions are published to Chocolatey with no way of checking the EAP versions of PyCharm at all.

## Possible Solution

@Spunkie has this set up already for our PHPStorm package (he's building the EAP releases and my repo only takes the final releases, don't ask me why we split it that way) over here: https://github.com/Spunkie/chocolatey-packages/tree/master/automatic/phpstorm

## Context

Jetbrains releases public Release Candidate versions of their products usually a couple of weeks before the final version to get feedback from the community and fix real-world issues.
They're definitely prerelease versions, but Chocolatey has a feature to distinguish final and prerelease versions and only offer prerelease versions to people explicitly asking for them.
Also JetBrains does the EAP thing very regularly with almost each of their products and almost every "feature" (in the year.feature.patch schema) version.

**Comment by mkevenaar** [comment]
@mastacheata I don't believe the current versions of PHPStorm supports EAP versions.

At this point I am currently not intended to implement this. If you want, you can always submit a PR.

**Comment by mastacheata** [comment]
> I don't believe the current versions of PHPStorm supports EAP versions.
Are you referring to the fact that there is currently no EAP version of PHPStorm on the Jetbrains website or that your packaging system doesn't support the prerelease flag of chocolatey?

PHPStorm EAP happens every few months or so. Right now there's no EAP version since they just released the 2020.2 versions of their IDEs one or two weeks ago.

As for your system of autoupdates not supporting prerelease flags => Well, that's why I asked and provided an example of how to do it based on the PHPStorm release I'm co-maintaining with user spunkie.
Basically we just have 2 maintainers and separate repositories and CD for the stable and prerelease versions.
